portal/portal-impl/impl/src/bundle/sitenav_eu.properties

```diff
@@ -13,6 +13,7 @@ log.userid=erabiltzaile IDa\:
 log.inputuserplaceholder=Erabiltzailea
 log.inputpasswordplaceholder=Pasahitza
 
+sit_role=Egungo Rola:
```

Comment:
Egungo Rola {0}.

@danielmerino Could you help me here please? haha

Reply:
I've asked Pablo. He thinks that the right basque text should be: 

Zure rola: {0}